### ISSUE TITLE
number of bbox

### ISSUE THREAD

**Issue by chuangchuangtan** [junsukchoe/ADL#10]
Thanks for sharing your code.I have a question about number of bbox.
why one image just one bbox in imagenet val dataset that you provide, it's different from imagenet that download from http://image-net.org?
Thank you

**Comment by junsukchoe** [comment]
As you mentioned, I only used one bounding box for validating our algorithm. This is because I just did not aware that there are multi object bounding boxes in ImageNet-1k dataset by mistake. Recently, I find that there are multiple bounding boxes in the dataset.

As far as I know, the localization accuracy metric of ImageNet localization challenge considers the prediction as correct when IoU between the estimated box and one of the ground-truth bounding boxes is over than 50%. 

So, unfortunately, I think that the accuracy in our algorithm is underestimated. I have a plan on releasing revised scores by arXiv, but it takes some times because of other on-going projects.
